pyomo/contrib/pynumero/interfaces/pyomo_nlp.py

```diff
@@ -170,7 +170,11 @@ def get_pyomo_inequality_constraints(self):
                 six.iteritems(self._condata_to_ineq_idx)}
         return [idx_to_condata[i] for i in range(len(idx_to_condata))]
 
+    @deprecated(msg='This method has been replaced with primals_names', version='', remove_in='6.0')
```

Comment:
Is your intent to remove this method for the next release?  If that is the case, I would recommend `version='6.0.0.dev0', remove_in='6.0'`.  Otherwise, I would either remove `remove_in`, or make it a later release (`remove_in` is the release that will no longer contain this method).

Reply:
done.